includes/admin/class-wc-admin-addons.php

```diff
@@ -18,7 +18,30 @@
 class WC_Admin_Addons {
 
 	/**
+	 * Get featured for the addons screen
+	 *
+	 * @return array of objects
+	 */
+	public static function get_featured() {
+		if ( false === ( $featured = get_transient( 'wc_addons_featured' ) ) ) {
+			$raw_featured = wp_safe_remote_get( 'https://dl.dropboxusercontent.com/u/2659230/wc/featured.json', array( 'user-agent' => 'WooCommerce Addons Page' ) );
```

Comment:
This is really comes from Dropbox?
Because no way for it works for all users. Dropbox will block for sure.